src/backend/src/routes/projects.routes.ts

```diff
@@ -35,10 +36,10 @@ projectRouter.post(
   body('otherConstraints').isArray(),
   body('otherConstraints.*.id').isInt({ min: -1 }).not().isString(),
   nonEmptyString(body('otherConstraints.*.detail')),
-  nonEmptyString(body('googleDriveFolderLink')),
-  nonEmptyString(body('slideDeckLink')),
-  nonEmptyString(body('bomLink')),
-  nonEmptyString(body('taskListLink')),
+  body('links').isArray(),
+  nonEmptyString(body('links.*.name')),
+  nonEmptyString(body('links.*.url')),
+  nonEmptyString(body('links.*.linkTypeId')),
```

Comment:
realistically we could probably have link type just be the name itself and make it @unique @id in the schema and that should work

no point in the link type id being a uuid which makes us add fields like this and stuff

Reply:
Thats true i didnt think about that